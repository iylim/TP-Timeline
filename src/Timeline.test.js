import React from 'react';
import { shallow } from 'enzyme';
import Timeline from './Timeline';

describe('Timeline', () => {
  it('should render', () => {
    const component = shallow(<Timeline debug />);
  
    expect(component).toMatchSnapshot();
  })
})