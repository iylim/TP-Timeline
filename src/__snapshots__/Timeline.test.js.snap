// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timeline should render correctly without props 1`] = `
<div
  className="vertical-timeline vertical-timeline--animate vertical-timeline--two-columns"
/>
`;
